frontend/src/app/rooms/[roomId]/page.js

```diff
@@ -61,6 +61,8 @@ function Page() {
     });
 
     socket.on("roundEnded", () => {
+      console.log(roundResult);
+      setRoundResult(roundResult);
```

Comment:
right now, this isn't updating roundResult with a new value. We need to do something like:

```
socket.on("roundEnded", ({updatedPlayers, roundPlayerRankings}) => {
      setPlayers(updatedPlayers);
      setRoundResult(roundPlayerRankings);
      setScreen(roundResultsScreen);
}
```

(reference: https://github.com/PlaylistCrafters/CS375Spotify/wiki/Socket-Game-Events#sendroundresults)